### PR TITLE
[aclorch] Create sonic-swss tests for dtel acl

### PR DESCRIPTION
**Pull request by KostiantynYarovyiBf**
**What I did**
Create sonic-swss tests for dtel acl
**Why I did it**
tests  are absent
**How I verified it**
run test_dtel.py::TestDtel::test_DtelFlowWatchlist by pytest
**Details if related**

### CODE REVIEW

**Comment by KostiantynYarovyiBf** [comment]
@mickeyspiegel  please review

File: tests/test_dtel.py [hunk 1]
```diff
@@ -211,7 +211,112 @@ def test_DtelQueueReportAttribs(self, dvs, testlog):
...
 
         tbl._del("Ethernet0|0")
-    
+
+    def test_DtelFlowWatchlist(self, dvs, testlog):
+        self.db = swsscommon.DBConnector(4, dvs.redis_sock, 0)
+        self.adb = swsscommon.DBConnector(1, dvs.redis_sock, 0)
+        self.table = "DTEL_FLOW_WATCHLIST"
+
+        fieled_1=[("PRIORITY", "30"),
```
Comment (mickeyspiegel) [inline tests/test_dtel.py]: nit: fieled -> fields

File: tests/test_dtel.py [hunk 2]
```diff
@@ -211,7 +211,112 @@ def test_DtelQueueReportAttribs(self, dvs, testlog):
...
+                 ("REPORT_ALL_PACKETS", "FALSE"),
+                 ("DROP_REPORT_ENABLE", "TRUE"),
+                 ("TAIL_DROP_REPORT_ENABLE", "TRUE")]
+        listfield = [fieled_1, fieled_2, fieled_3, fieled_4, fieled_5]
+
+        for field in listfield:
+            k = listfield.index(field)
+            rule = "RULE-" + str(k)
+            self._create_dtel_acl_rule(self.table, rule, field)
+            time.sleep(1)
```
Comment (mickeyspiegel) [inline tests/test_dtel.py]: There are a lot of time.sleep(1) in this code, probably more than are needed.

**Review by mickeyspiegel — APPROVED** [review]

File: tests/test_dtel.py (hunk 1 above)
Comment (KostiantynYarovyiBf) [reply tests/test_dtel.py]: done

File: tests/test_dtel.py (hunk 2 above)
Comment (KostiantynYarovyiBf) [reply tests/test_dtel.py]: done

**Comment by KostiantynYarovyiBf** [comment]
@mickeyspiegel 
I haven't permission to merge. Can you merge?

**Pull request merged**